utils/subpath.go

```diff
@@ -85,13 +85,13 @@ func UpdateAssetsSubpath(subpath string) error {
 
 	newRootHtml := string(oldRootHtml)
 
-	reCSP := regexp.MustCompile(`<meta http-equiv="Content-Security-Policy" content="script-src 'self' cdn.rudderlabs.com/([^"]*)">`)
+	reCSP := regexp.MustCompile(`<meta http-equiv="Content-Security-Policy" content="script-src 'self' cdn.rudderlabs.com/ js.stripe.com/v3([^"]*)">`)
```

Comment:
We may not need to add here given but only below in `web/handlers.go`, unless we see there is a possibility that in cloud the webapp may be hosted in a subpath.

Reply:
Good to keep IMO 👍 I could see us using subpaths in the future